Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/README.md

```diff
@@ -1055,4 +1055,49 @@ There are no input arguments for this command.
 
 #### Context Output
 
-There is no context output for this command.
\ No newline at end of file
+There is no context output for this command.
+### msgraph-get-managed-device-physical-memory-by-id
+
+***
+Get the managed device physical memory bytes.
+
+#### Base Command
+
+`msgraph-get-managed-device-physical-memory-by-id`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| device_id | The ID of the managed device to be fetched (Can be retreived using the msgraph-list-managed-devices command). | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| MSGraphDeviceManagement.Device.id | String | The ID of the managed device. | 
+| MSGraphDeviceManagement.Device.physicalMemoryInBytes | String | The correct value of ID of the managed device. | 
+| MSGraphDeviceManagement.Device.deviceName | String | Name of the device. | 
```

Comment:
Why do we need to return also the deviceName? So as for the ID. Just the physicalMemoryInBytes matters.

Reply:
in case the command is being executed before getting the device by ID, more details of the device